### PR TITLE
Update README and contribution pages

### PR DESCRIPTION
**Pull request by jzding**
Signed-off-by: Jack Ding <jacding@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,129 +1,93 @@
...
+```go
+package main
+import (
+	v1pubsub "github.com/redhat-cne/sdk-go/v1/pubsub"
+    v1amqp "github.com/redhat-cne/sdk-go/v1/amqp"
+	"github.com/redhat-cne/sdk-go/pkg/types"
+)
+func main(){
+    
+    resourceAddressHwEvent := "/cluster/node/nodename/redfish/event"
```
Comment (aneeshkp) [inline README.md:75]: `nodename` should be real node name form env

File: docs/development.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+export HW_PLUGIN=true; export HW_EVENT_PORT=9087; export CONSUMER_TYPE=HW
+export MSG_PARSER_PORT=9097; export MSG_PARSER_TIMEOUT=10
+export LOG_LEVEL=debug
+# replace the following with real Redfish credentials and BMC ip address
+export REDFISH_USERNAME=admin; export REDFISH_PASSWORD=admin; export REDFISH_HOSTADDR=127.0.0.1
+
+```
+
+### Install and run Apache Qpid Dispach Router
+```shell
```
Comment (aneeshkp) [inline docs/development.md:20]: you don't need AMQ Interconnect operator here?

Comment (jzding) [reply docs/development.md:20]: no. This is for local test, not in a cluster.

File: README.md (hunk 1 above)
Comment (jzding) [reply README.md]: updated

**Review by aneeshkp — APPROVED** [review]

**Pull request merged**